README.MD

```diff
@@ -1942,8 +1942,8 @@ This behavior is controlled by:
 ```kotlin
 aem {
     localInstance {
-        rootDir = prop.string("localInstance.root")?.let { project.file(it) } ?: projectMain.file(".aem/instance")
-        overridesDir = File(configCommonDir, LocalInstance.FILES_PATH)
+        rootDir = File(".instance/[author|publish]")
```

Comment:
this is confusing... because putting this values as;  documented will be not correct in all cases ; and you missed `local`

Reply:
maybe, I tried to make it simple, because, even knowing GAP a little from inside, I had difficulties understanding this documentation - in my opinion it is worth rephrasing, maybe we can have a call and work on it together?